src/main/java/com/regexr/ParseWeb.java

```diff
@@ -0,0 +1,67 @@
+package com.regexr;
+
+
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.log4j.Logger;
+
+
+public class ParseWeb {
+    private static final Logger LOG = Logger.getLogger(ParseWeb.class);
+    private static final String IP_ADDRESS_PATTERN = "((2[0-5][0-5]|1?[0-9]?[0-9])\\.){3}(2[0-5][0-5]|1?[0-9]?[0-9])";
+    private static final String EMAIL_PATTERN = "(\\b[\\w.-]+@[-.\\w]+\\b)";
+    private static final String NUM_PATTERN = "\\+?(\\d{2})?(0\\d{9})";
+    private static final String DATE_PATTERN = "(\\d{1,2}\\D{3,7}я\\d{4})";
+    private String numbers;
+    private String email;
+    private String ip;
+    private String date;
+    private Document doc;
+
+    public ParseWeb(String url) {
+        try {
+            doc = Jsoup.connect(url).get();
+        } catch (IOException e) {
+            LOG.info("Connection error");
```

Comment:
should be an error not info. info -> erro
it is not a very good idea to put logic into the constructor.